jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -985,6 +1003,7 @@ def dot_product_attention(query: Array,
     dropout_rate: Dropout rate.
     qkv_layout: Layout string, with supported formats being BTNH, BNTH, BSNH,
                 BNSH.
+    sliding_window_length: Sliding window attention length, default disabled.
```

Comment:
Would it be obvious to users what sliding window is referring to? I'm not familiar with the underlying CuDNN API, so I'm personally not sure what the window is. Perhaps, add more context in the docstring?